### ISSUE TITLE
 first time open admin panel was a little slow

### ISSUE THREAD

**Issue by DaZiYuan** [strapi/strapi#3191]
**Informations**
- **Node.js version**:v10.15.3
- **NPM version** : 6.4.1
- **Strapi version**: 3.0.0-alpha.26
- **Database**: mysql
- **Operating system**: ubuntu18
- **(Optional) Link to your Project**: https://w.mscoder.cn/admin/

**What is the current behavior?**
admin panel open wait too long

**Steps to reproduce the problem**
https://w.mscoder.cn/admin/
![image](https://user-images.githubusercontent.com/80653/56778741-7990b880-680a-11e9-94b0-624eecf301ae.png)

**What is the expected behavior?**
load faster

**Suggested solutions**

**Comment by aminsource** [comment]
I have same issue http://api.smarttools.ir

**Comment by soupette** [comment]
I agree, this issue will be solved with the beta. The way we load the plugin has been changed so it will load way faster.

**Comment by aminsource** [comment]
![Capture](https://user-images.githubusercontent.com/5094371/56803095-03ef1180-6837-11e9-8916-e993d52ac3ed.JPG)

**Comment by lauriejim** [comment]
Hello @DaZiYuan and @aminsource !
Thank you for this feedback. Appreciate.
As @soupette said, we are working on it for the beta.

I close this issue cause it's more an note that we are working on. Thank you!

**Comment by harenlewis** [comment]
Hey @lauriejim  any update on this or any workaround that we can do to reduce the initial load time?

**Comment by lauriejim** [comment]
Ping @soupette for this answer.

**Comment by soupette** [comment]
@harenlewis which version of Strapi are you using? The alpha version was really slow to load but now that we have reworked the load of the plugins in the administration panel I think that it loads faster. 
> Hey @lauriejim any update on this or any workaround that we can do to reduce the initial load time?

Are you able to identify what is slow (requests time,...)?

**Comment by harenlewis** [comment]
Hi @soupette apologize for the late reply.
Here's what my dependencies look like in the package.json

```
"dependencies": {
    "knex": "<0.20.0",
    "pg": "^8.0.1",
    "sqlite3": "latest",
    "strapi": "3.0.0-beta.19.3",
    "strapi-admin": "3.0.0-beta.19.3",
    "strapi-utils": "3.0.0-beta.19.3",
    "strapi-plugin-content-type-builder": "3.0.0-beta.19.3",
    "strapi-plugin-content-manager": "3.0.0-beta.19.3",
    "strapi-plugin-users-permissions": "3.0.0-beta.19.3",
    "strapi-plugin-email": "3.0.0-beta.19.3",
    "strapi-plugin-upload": "3.0.0-beta.19.3",
    "strapi-connector-bookshelf": "3.0.0-beta.19.3"
  }
```
Attaching the network details 
![Screenshot from 2020-06-05 00-40-31 (2)](https://user-images.githubusercontent.com/12905568/83802261-0b364f00-a6c8-11ea-9abe-38eb0d65aa3d.png)

Let me know if I can help with anything more!

**Comment by brenojac** [comment]
Hello guys,

Just to let you know that I'm running successfully a new strapi installation according to the documentation, using Docker Compose and MongoDB (https://strapi.io/documentation/3.0.0-beta.x/installation/docker.html) and the first time I load the admin interface takes a loooong time. More than a minute or so. Does that mean this issue has not been resolved in the beta version yet? 

best regards,

- Breno

**Comment by Moizsohail** [comment]
Has this issue been resolved? I am currently using 3.0.5 but the issue still persists on my end. It takes about 1.9 minutes for the chunk to fully download

**Comment by derrickmehaffy** [comment]
Hello!

Can you please create a new GitHub issue by using the GitHub issue template, providing all required information.

Your issue looks related to that topic but it's an old one. Using another thread (GitHub issue) will be able to manage your case.

Thank you and have a good day.
